src/libraries/math/util_math.cc

```diff
@@ -303,6 +303,7 @@ double adaptiveSimpsonsAux2(double (*f)(double x, void *p), void *arg,
                             double a, double b, double epsilon,
                             double S, double fa, double fb, double fc, int bottom)
 {
+  GEOTIMER_PREFIX(__func__);
```

Comment:
you can remove it, the time spent here is negligible.

Reply:
I put it just to know the n° of calls but the output is strange:
```
+---------------------------------------------+------------+------------+
| Total CPU time elapsed since start          |       8.9s |            |
|                                             |            |            |
| Section                         | no. calls |  CPU time  | % of total |
+---------------------------------+-----------+------------+------------+
| EnergyBalance                   |         1 |      4.97s |      55.8% |
| PointEnergyBalance              |      6781 |      4.96s |      55.8% |
| SolvePointEnergyBalance         |      6781 |      0.54s |       6.1% |
| adaptiveSimpsonsAux2            |   2221851 |     39.52s |     443.8% |
| get_all_input                   |         1 |      3.85s |      43.2% |
| sky_view_factor                 |         1 |      3.40s |      38.2% |
| write_output                    |         1 |      0.01s |      0.00% |
| write_snow_file                 |        24 |      0.00s |      0.00% |
+---------------------------------+-----------+------------+------------+
```

Reply:
**| adaptiveSimpsonsAux2            |   2221851 |     39.52s |     443.8% |**

Reply:
ok, done